### PR TITLE
Update Ink dependencies

### PR DESCRIPTION
**Pull request by nahue**
### What are the changes and their implications?
This updates ink, ink-link and ink-spinner versions.
There were some changes required, due to ink 3 deprecations listed in https://github.com/vadimdemedes/ink/blob/master/migrate.md

### Checklist

- [ ] Tests added for changes
- [ ] PR submitted to [blitzjs.com](https://github.com/blitz-js/blitzjs.com) for any user facing changes

### CODE REVIEW

**Comment by nahue** [comment]
There are some test errors that don't seem related to this PR.

**Comment by flybayer** [comment]
Thank you @nahue! Changes look good, but I think something here is definitely causing the test failure. It could be a test needs changed, but either way we need them to pass before merging :)

**Comment by flybayer** [comment]
@nahue can you investigate these test failures?

**Comment by nahue** [comment]
> @nahue can you investigate these test failures?

Yes!

**Comment by nahue** [comment]
> @nahue can you investigate these test failures?

I pulled upstream into this branch and now its passing.

File: README.md [hunk 1]
```diff
@@ -401,6 +401,7 @@ Thanks to these wonderful people ([emoji key](https://allcontributors.org/docs/e
 
 <!-- markdownlint-enable -->
 <!-- prettier-ignore-end -->
+
```
Comment (nahue) [inline README.md:404]: This is being introduced by the husky hook... i dont use formatOnSave :D

**Comment by flybayer** [comment]
Thanks!! 

@all-contributors add @nahue for code

**Pull request merged**